tests/sdk/nodejs/examples/examples_test.go

```diff
@@ -182,13 +182,6 @@ func TestAccHelm(t *testing.T) {
 					}
 				}
 			},
-			EditDirs: []integration.EditDir{
```

Comment:
Why do the two `step2` disappear?

Reply:
Those tests were checking that the Chart could be created with the `v2` SDK, and subsequently updated to `v3`. Since `v2` is gone, the second step is no longer needed.